frontend/src/components/CreateForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useListingsContext } from '../hooks/useListingsContext';
+
+const CreateForm = (props) => {
+    const { dispatch } = useListingsContext();
+    const id = localStorage.getItem('id');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const { title, organisationName, description, requirement, firstLine, city, postcode, neededByDate } = e.target.elements;
+        const listing = {
+            title: title.value,
+            organisationName: organisationName.value,
```

Comment:
how difficult would it be to implement a way for user not to have to type this themselves but is fetched from their auth state through their user id, this will be needed for other listing functionality

if its a long job, then maybe merge and start in a new ticket

Reply:
Merged main in to my branch and have it working. Only thing that seems to be causing me grief is trying to get the orgname to auto populate. I've tried a few things but keep ending up with errors.